media/css/base/mozilla-modal.less

```diff
@@ -70,6 +70,7 @@ html.noscroll body {
                     .image-replaced();
                     border-radius: 50%;
                     border: 2px solid #fff;
+                    transition: transform 0.15s ease;
```

Comment:
Personally I think Webkit is a bit of a different issue to evergreen desktop browsers. There are so many mobile webkit devices out there that are not updated, it doesn't seem like much hardship to keep the last vendor prefix in there. But I could be persuaded that we're not too concerned about old devices...